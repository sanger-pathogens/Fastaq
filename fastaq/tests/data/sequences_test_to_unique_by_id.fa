>seq1
AA
>seq2
A
>seq3
A
>seq1
A
>seq4
>seq1
AAA
