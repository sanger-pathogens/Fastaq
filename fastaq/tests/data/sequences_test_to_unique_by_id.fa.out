>seq1
AAA
>seq2
A
>seq3
A
